src/gmp_get.c

```diff
@@ -63,37 +65,33 @@ get_data_parse_attributes (get_data_t *data, const gchar *type,
 
   data->type = g_strdup (type);
 
-  append_attribute (attribute_names, attribute_values, "filter",
-                    &data->filter);
+  append_attribute (attribute_names, attribute_values, "filter", &data->filter);
 
   name = g_strdup_printf ("%s_id", type);
-  append_attribute (attribute_names, attribute_values, name,
-                    &data->id);
+  append_attribute (attribute_names, attribute_values, name, &data->id);
   g_free (name);
 
-  append_attribute (attribute_names, attribute_values, "filt_id",
-                    &data->filt_id);
+  append_attribute (
+    attribute_names, attribute_values, "filt_id", &data->filt_id);
 
-  if (find_attribute (attribute_names, attribute_values,
-                      "trash", &attribute))
+  if (find_attribute (attribute_names, attribute_values, "trash", &attribute))
     data->trash = strcmp (attribute, "0");
   else
     data->trash = 0;
 
-  if (find_attribute (attribute_names, attribute_values,
-                      "details", &attribute))
+  if (find_attribute (attribute_names, attribute_values, "details", &attribute))
     data->details = strcmp (attribute, "0");
   else
     data->details = 0;
 
-  if (find_attribute (attribute_names, attribute_values,
-                      "ignore_pagination", &attribute))
+  if (find_attribute (
+        attribute_names, attribute_values, "ignore_pagination", &attribute))
     data->ignore_pagination = strcmp (attribute, "0");
   else
     data->ignore_pagination = 0;
 
-  append_attribute (attribute_names, attribute_values, "filter_replace",
-                    &data->filter_replace);
+  append_attribute (
```

Comment:
I find this style such a weird choice.  Putting the args one per line after the paren would be
more readable for me.  Maybe Clang will add this option in the future.